databases/dbeaver-community/PortFile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   java 1.0
+
+github.setup                dbeaver dbeaver 6.1.4
+github.tarball_from         releases
+distname                    dbeaver-ce-${version}-macosx.cocoa.x86_64
+name                        dbeaver-community
```

Comment:
Why dbeaver-community, and not just dbeaver ?

Reply:
There's also an Enterprise Edition with a separate download, non-OSS license and it's own homepage (dbeaver.io vs. dbeaver.com). Unlike e.g. dbvis, which comes as a single download and can be run in Free and Pro modes, the DBeaver Community Edition cannot be switched to Enterprise mode by entering a license. 

"dbeaver-community" is also the name of it's [homebrew cask](http://macappstore.org/dbeaver-community/), so I just copied that for consistency. :-)

Reply:
OK, I see. that seems fine then.